main.py

```
@@ -200,29 +226,39 @@ def zip_sb3(fnc, project_dir):
-        jsonfile, fnc = make_filenames(p, project, translation_table)
-
-        # Download and zip the zb3
-        download = download_sb3(project, fnc, jsonfile)
-        if not download:
-            continue
-        project_dir = download
+    while True:
+        pagenum = 1
+        try: 
+            projects = session.mystuff_projects(choice, page=pagenum, sort_by="")
+        except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)